tools/deb/istio-iptables.sh

```diff
@@ -46,7 +46,9 @@ function usage() {
   echo '  -x: Comma separated list of IP ranges in CIDR form to be excluded from redirection. Only applies when all '
   # shellcheck disable=SC2016
   echo '      outbound traffic (i.e. "*") is being redirected (default to $ISTIO_SERVICE_EXCLUDE_CIDR).'
+  echo '  -c: Enable core dump. Specify the core pattern as the argument (default is /var/lib/istio/core.proxy)'
```

Comment:
I'd recommend against introducing a default value here. Better to force the caller to specify to ensure a single source of truth.